File: examples/typescript/policies/listPolicies.ts
// Usage: pnpm tsx policies/listPolicies.ts

import { CdpClient } from "@coinbase/cdp-sdk";
import "dotenv/config";

const cdp = new CdpClient();
const policies = await cdp.policies.listPolicies();
console.log("Listed policies: ", JSON.stringify(policies, null, 2));

File: examples/python/policies/list_policies.py
# Usage: uv run python policies/list_policies.py

import asyncio

from cdp import CdpClient
from dotenv import load_dotenv

load_dotenv()

async def main():
    async with CdpClient() as cdp:
        policies = await cdp.policies.list_policies()
        print("Listed policies: ", policies)


asyncio.run(main())
